consensus/config/src/committee.rs

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use multiaddr::Multiaddr;
+use serde::{Deserialize, Serialize};
+
+use crate::{NetworkPublicKey, ProtocolPublicKey};
+
+/// Committee of the consensus protocol is updated each epoch.
+pub type Epoch = u64;
+
+/// Each authority is uniquely identified by its AuthorityIndex in the Committee.
+/// AuthorityIndex is between 0 (inclusive) and the total number of authorities (exclusive).
+pub type AuthorityIndex = u32;
+
+/// Voting power of an authority, roughly proportional to the actual amount of Sui staked
+/// by the authority.
+/// Total stake / voting power of all authorities should sum to 10,000.
+pub type Stake = u64;
+
+/// Network information of one authority in the committee.
+#[derive(Serialize, Deserialize)]
+pub struct NetworkInfo {
+    /// Network address for communicating with the authority.
+    pub address: Multiaddr,
+    /// The validator's hostname, for metrics and logging.
+    pub hostname: String,
+    /// The authority's ed25519 publicKey for signing network messages and blocks.
+    pub network_key: NetworkPublicKey,
+    /// The authority's bls public key for random beacon.
```

Comment:
This will only be used for random beacon?

Reply:
Exactly. Aggregated signatures will not be on the critical path in consensus 2.0.